src/index.html

```diff
@@ -129,12 +146,63 @@ <h2 class="modal__title">New Post</h2>
     </div>
     <div class="modal" id="preview-modal">
       <div class="modal__content modal__content_type_preview">
-        <button aria-label="close modal" type="button" class="modal__close-button modal__close-button_type_preview"></button>
-        <img class="modal__image" src="./images/2-photo-by-ceiline-from-pexels.jpg" alt="image"/>
+        <button
+          aria-label="close modal"
+          type="button"
+          class="modal__close-button modal__close-button_type_preview"
+        ></button>
+        <img class="modal__image" src="#" alt="image" />
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script src="./scripts/validation.js"></script>
-    <script src="./scripts/index.js"></script>
+    <div class="modal modal_type_image">
+      <div class="modal__content">
+        <img class="modal__image" src="" alt="" />
+        <p class="modal__caption"></p> 
+        <button class="modal__close"></button>
+      </div>
+    </div>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-button"></button>
+        <h2 class="modal__title">Edit Profile Avatar</h2>
+        <form class="modal__form-avatar" id="edit-avatar-form">
+          <label for="profile-avatar-input" class="modal__label">
+            Profile Picture
+            <input
+              id="profile-avatar-input"
+              type="url"
+              class="modal__input"
+              name="avatar"
+              placeholder="Paste a link to an image"
+              required
+            />
+            <span id="profile-avatar-input-error" class="modal__error">
+              Please enter a URL
+            </span>
+          </label>
+          <button type="submit" class="modal__submit-button">Save</button>
+        </form>
+      </div>
+    </div>
+    <div class="modal" id="delete-modal">
```

Comment:
* Please, make the `confirmation` modal match the design: `desktop` and `mobile`